packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -96,10 +96,15 @@
           "when":
             "sfdx:project_opened && sfdx:replay_debugger_checkpoints_enabled"
         },
+        {
+          "command": "sfdx.toggle.checkpoint",
+          "when":
+            "sfdx:project_opened && sfdx:replay_debugger_checkpoints_enabled && editorLangId==apex"
```

Comment:
In other places, we have spaces before and after `==`, and the language ID is in single quotes.

Reply:
It will be done.